drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -1174,6 +1209,7 @@ local matter_driver_template = {
         [clusters.ElectricalPowerMeasurement.attributes.ActivePower.ID] = active_power_handler,
       },
       [clusters.ElectricalEnergyMeasurement.ID] = {
+        [clusters.ElectricalEnergyMeasurement.attributes.CumulativeEnergyImported.ID] = energy_report_handler_factory(true),
```

Comment:
sorry for the confusion- see [this PR](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1830) which will change the system to only use imported and not exported energy. This fixup be merged before this PR.